packages/e2e-tests/src/specs/editor/shopping/productMenu.js

```diff
@@ -27,7 +27,7 @@ import {
 jest.retryTimes(3, { logErrorsBeforeRetry: true });
 
 describe('Shopping product', () => {
-  minWPVersionRequired('5.8');
+  minWPVersionRequired('5.9');
```

Comment:
The idea on the ticket was to actually completely **remove** all these checks.

We added them because WooCommerce requires 5.8+ but we still tested on 5.7

Now that we test on 5.9+, these checks are superfluous.